src/Microsoft.DotNet.Darc/Darc/Helpers/LocalSettings.cs

```diff
@@ -48,18 +48,10 @@ private static LocalSettings LoadSettingsFile(ICommandLineOptions options)
         {
             return LoadSettingsFile();
         }
-        catch (Exception exc) when (exc is DirectoryNotFoundException || exc is FileNotFoundException)
+        catch (Exception e) when (e is DirectoryNotFoundException || e is FileNotFoundException)
```

Comment:
The `options` parameter is no longer in use. I think we can combine `LoadSettingsFile()` and `LoadSettingsFile(ICommandLineOptions options)`
```
    private static LocalSettings LoadSettingsFile()
    {
        try
        {
        string settings = EncodedFile.Read(Constants.SettingsFileName);
        return JsonConvert.DeserializeObject<LocalSettings>(settings);
        }
        catch (Exception e) when (e is DirectoryNotFoundException || e is FileNotFoundException)
        {
        return null;
         }
```
and add a warning record that a directory or a file does not exist.

Reply:
Yeah, nice simplification.

I cannot add the warning because it is fine to not have the file.